aws_lambda_builders/utils.py

```diff
@@ -222,3 +223,32 @@ def extract_tarfile(tarfile_path: Union[str, os.PathLike], unpack_dir: Union[str
                 raise tarfile.ExtractError("Attempted Path Traversal in Tar File")
 
         tar.extractall(unpack_dir)
+
+
+def glob_copy(source: Union[str, list], destination: str) -> None:
+    items = source if isinstance(source, list) else [source]
+    dest_path = Path(destination)
+    known_paths = []
+    for item in items:
+        if Path(item).is_absolute():
+            raise ValueError('"{item}" is not a relative path'.format(item=item))
```

Comment:
Let's use custom exceptions instead of raising `ValueError`.

Reply:
Any specific one you want me to use, or should I roll one?